src/com/xilinx/rapidwright/tests/CodePerfTracker.java

```diff
@@ -280,6 +285,15 @@ public boolean isUsingGCCallsToTrackMemory() {
      */
     public void useGCToTrackMemory(boolean useGCCalls) {
         this.trackMemoryUsingGC = useGCCalls;
+        setTrackOSMemUsage(true);
+    }
+    
+    /**
+     * Sets tracking of OS memory usage (in Linux only).
+     * @param trackOSMemUsage If true, will track curr and peak memory usage of the process as 
+     * reported by the OS. 
+     */
+    public void setTrackOSMemUsage(boolean trackOSMemUsage) {
```

Comment:
Oh, looks like `trackOSMemUsage` is not actually used here!

Reply:
Sorry, that's embarrassing!  Fixed.